0.5/Dockerfile

```diff
@@ -59,8 +59,11 @@ ENV JANUS_VERSION=${JANUS_VERSION} \
 
 RUN groupadd -r janusgraph --gid=999 && \
     useradd -r -g janusgraph --uid=999 janusgraph && \
+    printf "\\ndeb http://ppa.launchpad.net/rmescandon/yq/ubuntu xenial main\\n\
+deb-src http://ppa.launchpad.net/rmescandon/yq/ubuntu xenial main" >> /etc/apt/sources.list.d/janusgraph-dep.list
+RUN    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 9A2D61F6BB03CED7522B8E7D6657DBE0CC86BB64 && \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/manual/JanusGraph/janusgraph-docker/pullRequest?prid=5303974)